tests/Feature/Posts/UpdatePostsTest.php

```diff
@@ -66,5 +58,118 @@ public function it_does_allow_auth_user()
             'content' => $post->content,
             'status_id' => $post->status_id
         ]);
+
+        $this->assertDatabaseHas('posts', [
+            'id' => $post->id,
+            'name' => $post->name,
+            'content' => $post->content,
+            'status_id' => $post->status_id
+        ]);
+    }
+
+    /** @test */
+    public function it_resets_published_at_when_post_is_updated_to_draft()
```

Comment:
These are good tests although since the behaviour that they are testing is happening in the observer I would move them into a model test